cmd/service/init.go

```diff
@@ -43,6 +48,13 @@ func init() {
 	Init.Flags().StringP("description", "d", "", "Description")
 	Init.Flags().BoolP("current", "c", false, "Create the service in the current path")
 	Init.Flags().StringP("template", "t", "", "Specify the template URL to use")
+
+	// TODO(ilgooz): remove this after service package made Newable.
+	c, err := container.New()
+	if err != nil {
+		log.Fatal(err)
+	}
+	defaultContainer = c
```

Comment:
I think this is to initialize a `defaultContainer` in the CLI but this file is the file for the command `mesg-core service init` so that might not make sense to put it here but maybe more in the `utils` where it's used

Also if it's only used one time in the `service/utils#buildDockerImage` why not initialize it in this function directly

Reply:
We are going to remove these lines after service package refactored. `*Container` will be initialized once and stored in the `*Service` struct. I don't mind where to put this temporary code, want me to move to utils for now?

Reply:
yes that would be good, I'm afraid to forget it here because it's not really related to this command